arm9/src/main.c

```diff
@@ -73,7 +73,7 @@ THUMB_FUNC void NitroMain(void)
     FUN_02002C50(1, 3);
     FUN_02002C50(3, 3);
     gBacklightTop.unk18 = -1;
-    gBacklightTop.unk20 = FUN_0202254C();
+    gBacklightTop.unk20 = SaveBlock2_new();
```

Comment:
I'm unsure if this should even be called gBacklightTop anymore, it clearly contains a lot more i.e. save block 2

Reply:
Imma leave it until we decide for sure what to call it.

Reply:
ok